doc/ref_model/chapters/chapter09.md

```diff
@@ -214,6 +214,31 @@ All phases described above can be automated using technology specific toolsets a
 <a name="9.5.3.1"></a>
 #### 9.5.3.1. Pre-tenant Creation Requirements
 
+Topics include:
+1. Tenant Approval -- use, capacity, data centers, ..
+    - Validate that the [Tenant's](../../../common/glossary.md#operational-and-administrative-terminology) planned use meets the Operators Cloud Use policies
+    - Validate that the capacity available within the requests cloud site(s) can satisfy the Tenant requested quota for vCPU, RAM, Disk, Network Bandwidth
+    - Validate that the Cloud Infrastructure can meet Tenant's performance requirements (e.g. I/O, latency, jitter, etc)
+    - Validate that the Cloud Infrastructure can meet Tenant's resilience requirements
+1. For VM-based environments:
+    - Verify that any requested private flavours have been created
+    - Verify that the metadata for these private flavours have been created
+    - Verify that the tenant has permissions to use the requested private flavours
+    - Validate that host aggregates are available for specified flavors (public and private)
+    - Verify that the metadata matches for the requested new flavours and host aggregates
+1. Tenant Networks
+    - Verify that the networks requested by the tenant exist
+    - Verify that the security policies are correctly configured to only approved ingress and egress
+1. Tenant Admin, Tenant Member and other Tenant Role approvals for user by role
+    - Add all Tenant Members and configure their assigned roles in the Enterprise Identity and Access management system (e.g., LDAP)
+    - Verify that these roles have been created for the Tenant
+1. Tenant Images and manifests approvals
```

Comment:
The verification of the images doesn't necessarily need to be executed pre-tenant creation. The tenant may later bring additional images etc or update images, which need to be checked as well.
Also this part is VM specific and we need similar text for containers.

Reply:
@ulikleber Thanks. Made a slight edit -- please review.

Reply:
I am still not sure, image verification is to be done **pre-tenant creation**. Or maybe I misunderstood what pre-tenant creation or "tenant image" means.